Xamarin.Forms.Platform.iOS/Material/MaterialButtonRenderer.cs

```diff
@@ -94,9 +86,20 @@ protected override void OnElementChanged(ElementChangedEventArgs<Button> e)
 			}
 		}
 
+		protected virtual ButtonScheme CreateButtonScheme()
+		{
+			return new ButtonScheme
+			{
+				TypographyScheme = new TypographyScheme(),
+				ColorScheme = MaterialColors.Light.CreateColorScheme()
+			};
+		}
+
 		protected override MButton CreateNativeControl()
 		{
-			return new MButton();
+			var button = new MButton();
+			ContainedButtonThemer.ApplyScheme(CreateButtonScheme(), button);
```

Comment:
This `ContainedButtonThemer` represents the "normal" button of a solid background. There are variants, no background/text, outline button, FAB. These are no longer individual/separate controls.